### PR TITLE
Fix bulkDelete on entites that are not in default locale

### PR DESCRIPTION
**Pull request by petersg83**

### CODE REVIEW

File: packages/strapi-plugin-i18n/services/entity-service-decorator.js [hunk 1]
```diff
@@ -26,7 +35,10 @@ const wrapParams = async (params = {}, ctx = {}) => {
     };
   }
 
-  if (has('id', params) && SINGLE_ENTRY_ACTIONS.includes(action)) {
+  if (
+    (paramsContain('id', params) && SINGLE_ENTRY_ACTIONS.includes(action)) ||
```
Comment (alexandrebodin) [inline packages/strapi-plugin-i18n/services/entity-service-decorator.js:39]: can you put those in vars to make the if state more understandable

**Review by Muhammadariesta — APPROVED** [review]
foo@bar.com

**Review by alexandrebodin — APPROVED** [review]

**Review by Muhammadariesta — APPROVED** [review]
Gmail

**Comment by petersg83** [comment]
Hi @Muhammadariesta, what are your comments for?

**Pull request merged**